components/bsx/Offer/OfferTable.vue

```diff
@@ -82,11 +82,11 @@
         field="expirationBlock"
         :label="$t('offer.expiration')"
         sortable>
-        <b-tooltip
+        <NeoTooltip
           v-if="!isExpired(props.row.expiration)"
           :label="calcExpirationDate(props.row.expiration)">
           {{ calcExpirationTime(props.row.expiration) }}
-        </b-tooltip>
+        </NeoTooltip>
         <span v-else>
           {{ calcExpirationTime(props.row.expiration) }}
         </span>
```

Comment:
should be able to use active prop on NeoTooltip to avoid the v-if/v-else

Reply:
it is set to true:

```ts
export interface Props {
  label: string | LocaleMessage
  position?: 'top' | 'bottom' | 'left' | 'right'
  active?: boolean
  multiline?: boolean
  appendToBody?: boolean
  delay?: number
  fontSize?: string | number
}
const props = withDefaults(defineProps<Props>(), {
  position: 'top',
  active: true,
  multiline: false,
  appendToBody: false,
  delay: undefined,
  fontSize: '1rem',
})```

Reply:
nvm my comment hehe just saw it's not set by default on purposes